docs/deprecated/clusters/public-cloud/azure/required-permissions.md

```diff
@@ -0,0 +1,967 @@
+            "Microsoft.Network/loadBalancers/inboundNatRules/delete",
+            "Microsoft.Network/loadBalancers/inboundNatRules/join/action",
+            "Microsoft.Network/loadBalancers/inboundNatRules/read",
+            "Microsoft.Network/loadBalancers/inboundNatRules/write",
+            "Microsoft.Network/loadBalancers/read",
+            "Microsoft.Network/loadBalancers/write",
+            "Microsoft.Network/networkInterfaces/delete",
+            "Microsoft.Network/networkInterfaces/join/action",
+            "Microsoft.Network/networkInterfaces/read",
+            "Microsoft.Network/networkInterfaces/write",
+            "Microsoft.Network/networkSecurityGroups/read",
+            "Microsoft.Network/networkSecurityGroups/securityRules/delete",
+            "Microsoft.Network/networkSecurityGroups/securityRules/read",
+            "Microsoft.Network/networkSecurityGroups/securityRules/write",
+            "Microsoft.Network/privateDnsZones/A/delete",
+            "Microsoft.Network/privateDnsZones/A/read",
+            "Microsoft.Network/privateDnsZones/A/write",
+            "Microsoft.Network/privateDnsZones/delete",
+            "Microsoft.Network/privateDnsZones/read",
+            "Microsoft.Network/privateDnsZones/virtualNetworkLinks/delete",
+            "Microsoft.Network/privateDnsZones/virtualNetworkLinks/read",
+            "Microsoft.Network/privateDnsZones/virtualNetworkLinks/write",
+            "Microsoft.Network/privateDnsZones/write",
+            "Microsoft.Network/publicIPAddresses/delete",
+            "Microsoft.Network/publicIPAddresses/join/action",
+            "Microsoft.Network/publicIPAddresses/read",
+            "Microsoft.Network/publicIPAddresses/write",
+            "Microsoft.Network/routeTables/delete",
+            "Microsoft.Network/routeTables/read",
+            "Microsoft.Network/routeTables/write",
+            "Microsoft.Network/virtualNetworks/join/action",
+            "Microsoft.Resources/subscriptions/resourceGroups/read",
+            "Microsoft.Network/virtualNetworks/read",
+            "Microsoft.Network/virtualNetworks/subnets/join/action",
+            "Microsoft.Network/virtualNetworks/subnets/read",
+            "Microsoft.Compute/galleries/images/read",
+            "Microsoft.Compute/galleries/images/versions/read"
+          ],
+          "AssignableScopes": ["/subscriptions/$SUBSCRIPTION_ID"]
+        }
+        ```
+    </details>
+
+3.  Create a role for the JSON file you created in the previous step. Issue the following command to create the role.
+
+    ```shell
+    az role definition create --role-definition @static-placement-permissions.json --output table
+    ```
+
+4.  Export the
+    [security principal](https://learn.microsoft.com/en-us/azure/role-based-access-control/overview#security-principal)
+    object ID and resource group name you want to use with Palette to a set of variables.
+
+    ```shell
+    export ASSIGNEE="<security_principal_object_id>"
```

Comment:
this needs to say `<security_principal_client_id>` instead of `<security_principal_object_id>`